src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/yml/messenger_routing.yml

```diff
@@ -2,5 +2,7 @@ framework:
     messenger:
         routing:
             'Symfony\Component\Messenger\Tests\Fixtures\DummyMessage': amqp
-            'Symfony\Component\Messenger\Tests\Fixtures\SecondMessage': [amqp, audit, ~]
```

Comment:
We need to keep a test that only has the array.

Reply:
done